website/server/controllers/api-v3/tasks.js

```diff
@@ -331,11 +335,12 @@ api.scoreTask = {
           user: user.profile.name,
           taskName: task.text,
         }),
+        groupId: group._id,
       });
 
       await Bluebird.all([groupLeader.save(), task.save()]);
 
-      return res.respond(200, {message: res.t('taskApprovalHasBeenRequested'), task});
```

Comment:
Yea, I had this as a 200 at first, but that didn't make sense. The error has some benefits on as well. Not sure what is best though.